### DISCUSSION TITLE
How to enable alternate web url for those of us using cloudflared?

### DISCUSSION THREAD

**Discussion by bobloadmire** [mrlt8/docker-wyze-bridge#669] [Q&A]
I can access my cams individually via cams.cloudflaredurl.com/name_of_cam fine, because i've set up port 8889 in cloudflared. however when working remotely the DWB page uses the internal URLs which obviously won't work. Is there a way to config DWB to over ride the base url with a custom one?

**Comment by mrlt8** [comment]
Yes, see https://github.com/mrlt8/docker-wyze-bridge#web-ui

You should be able to remap the HLS url with:

```yaml
- WB_HLS_URL=http://cams.cloudflaredurl.com/
```
